src/llm/http_llm_calculator.cc

```diff
@@ -536,6 +537,9 @@ class HttpLLMCalculator : public CalculatorBase {
             if (!this->request->isStream()) {
                 OVMS_PROFILE_SCOPE("Unary generation cycle");
                 std::vector<GenerationOutput> generationOutput = this->generationHandle->read_all();
+                std::sort(generationOutput.begin(), generationOutput.end(), [=](GenerationOutput& r1, GenerationOutput& r2) {
+                    return r1.score > r2.score;
+                });
                 RET_CHECK(generationOutput.size() >= 1);
```

Comment:
Maybe let's move this check above sorting. Functionally it doesn't make a difference, but it looks weird when we first use and then check the vector.